source/MaterialXShaderGen/ShaderGenerators/Osl/OslShaderGenerator.cpp

```diff
@@ -23,87 +23,87 @@ namespace
         "}\n\n";
 }
 
-const string OslShaderGenerator::LANGUAGE = "osl";
+const string OslShaderGenerator::LANGUAGE = "sx-osl";
```

Comment:
I decided to use "sx-osl" and "sx-glsl" as the new language names (compared to "osl-sx" and "glsl-sx"), mainly because the files and folder structures becomes a lot nicer and organized when sorted on "sx" instead of spread out.

Reply:
Guess ignore my earlier query about naming as this answers that one :)